build.savant

```diff
@@ -50,11 +51,12 @@ project(group: "org.savantbuild.plugin", name: "idea", version: "2.0.0-RC.7", li
 }
 
 // Plugins
-dependency = loadPlugin(id: "org.savantbuild.plugin:dependency:2.0.0-RC.6")
-groovy = loadPlugin(id: "org.savantbuild.plugin:groovy:2.0.0-RC.6")
-groovyTestNG = loadPlugin(id: "org.savantbuild.plugin:groovy-testng:2.0.0-RC.6")
-idea = loadPlugin(id: "org.savantbuild.plugin:idea:2.0.0-RC.7")
-release = loadPlugin(id: "org.savantbuild.plugin:release-git:2.0.0-RC.6")
+def dependency = loadPlugin(id: "org.savantbuild.plugin:dependency:2.0.0-RC.7")
```

Comment:
Same comment as the `savant-dependency-management` repo regarding the use of `def`.

Reply:
Reverted after talking w/ Daniel.